src/reference/faq.rst

```diff
@@ -83,3 +85,34 @@ For those who really want to use the ``arm64`` Miniconda installer, it is still
    conda config --env --set subdir osx-64
 
 This will ensure that all commands in the active Conda environment are run using ``osx-64`` emulation, making it possible to install all the software required for Nextstrain to run.
+
+.. _what-happened-to-the-native-runtime:
+
+What happened to the ``native`` runtime?
+----------------------------------------
+
+Up until Nextstrain CLI version 5.0.0, the name of the ``native`` runtime has been debated internally. In CLI version 5.0.0, **it was renamed to** ``ambient`` and we will use the new name going forwards.
+
+"Native" as a software term is typically used to describe software that can run without emulation, in other words optimized for your computer's processor.
+
+The ``ambient`` runtime is "native" in that sense, but it puts all the software maintenance burden on the user. This means:
+
+1. There is a lengthy setup process which requires installing external software (Conda, Mamba). Additionally, there is no way for us to provide accurate setup steps for users who already have Conda installed, as there are various methods of installing Conda.
+2. It is up to you as the creator of the ``nextstrain`` Conda environment to know (1) how to activate it, (2) when to update it, and (3) how to update it.
+
+So really, the ``ambient`` runtime is any environment that has been set up with all of the required software available on your local ``PATH``. We chose Conda in the installation instructions since some users may already be familiar with it, and it is simpler than using individual package managers for the various required software (e.g. ``pip``, ``npm``).
+
+Most importantly, Nextstrain CLI version 5.0.0 provides a **new** ``conda`` **runtime that runs natively** without putting all of the software maintenance burden on users. This means the ``ambient`` runtime is no longer the only "native" runtime, and we will recommend new users to use the ``conda`` runtime instead of ``ambient``.
+
+``ambient`` is still a good option for users who wish to customize their environment to include other software used in their workflows.
```

Comment:
I don't think this PR is ready, since the wording assumes that Conda runtime installation instructions are available when they aren't yet.

Reply:
Ah, I didn't read it that way, though I see what you mean.

> and we will recommend new users to use the ``conda`` runtime instead of ``ambient``.

I think this is true (i.e. we _will_ do it in the future), even though we're not quite at that point yet.

It seems ok to merge with this to me, but if you have revised wording you'd like, happy to go with that too.

Reply:
Oh I see, yeah this should be fine then. I'll merge.